src/ioc.ts

```diff
@@ -2,10 +2,12 @@ import { IocContainer } from '@tsoa/runtime'
 import { container } from 'tsyringe'
 
 import { logger, Logger, type ILogger } from './logger'
+import Database from './models/db/index.js'
 
 container.register<ILogger>(Logger, {
   useValue: logger,
 })
+container.register(Database, { useValue: new Database() })
```

Comment:
```suggestion
container.register<IDatabase>(Database, { useValue: new Database() })
```
not sure if we have an export of typeof Database = IDatabase

Reply:
Not necessary because this is a class. tsyringe can use the class as the specifier